### ISSUE TITLE
LayoutLM QA and BLIP decoder fail during ONNX export

### ISSUE THREAD

**Issue by fangyangci** [microsoft/winml-cli#1391]
## Description

Two Hugging Face model/task combinations fail during ONNX build/export with the current Transformers stack:

- `impira/layoutlm-document-qa` / `question-answering`
- `Salesforce/blip-image-captioning-base` / `image-to-text`

## Current behavior

- LayoutLM QA is loaded through `AutoModelForQuestionAnswering`, which rejects `LayoutLMConfig`. Loading the concrete class then exposes invalid random `token_type_ids` for checkpoints with `type_vocab_size=1`.
- The BLIP decoder wrapper passes a 4D additive attention mask, while the current HF BLIP implementation expects a 2D/3D binary mask and fails shape validation during export.

Representative errors:

```text
Unrecognized configuration class LayoutLMConfig for AutoModelForQuestionAnswering
index out of range in self
Wrong shape for input_ids [1,1] or attention_mask [1,1,1,512]
```

## Expected behavior

Both model/task combinations should export successfully and run with `NvTensorRTRTXExecutionProvider` on GPU.

## Environment

- Python 3.11.14
- Transformers 4.57.6
- NvTensorRTRTXExecutionProvider 2.30.43.0
